vocab/credentials/vocab.csv

```diff
@@ -5,6 +5,8 @@ odrl,prefix,,http://www.w3.org/ns/odrl/2/,,,,,
 xsd,prefix,,http://www.w3.org/2001/XMLSchema#,,,,,
 ,rdfs:seeAlso,,https://www.w3.org/TR/vc-data-model/,,,,,
 JsonSchemaValidator2018,rdfs:Class,,,,,,,A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+JsonSchema2023,rdfs:Class,,,,,,,A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+CredentialSchema2023,rdfs:Class,,,,,,,A type of validator that can be used to syntactically validate JSON documents packaged as Verifiable Credentials using the JSON Schema language.
```

Comment:
good flag. simply put, the spec operates on portions of a VC which it always treats as JSON, despite the outer wrapper being `vc+ld+json`. If there's a way I can make this more clear I'm open to suggestions.

Reply:
My suggestion is to just define ONE type for the schema validator you're defining in the JSON Schema 2023 specification. My suggestion is we just define "JsonSchema2023" in the v2 vocabulary, and any properties used therein, and call it a day. 

If it would be helpful, I could just rework the PR to do that, as I'm pretty sure I know what you're trying to accomplish in this PR. I don't want to make those changes to this PR w/o your permission, though. So, let me know if you want me to clean up this PR and I'll be happy to do so.

Reply:
I have no problem with you making these changes